Examples/Sources/MasterDetailTableViewController.swift

```diff
@@ -237,20 +237,84 @@ class MasterDetailTableViewController: UITableViewController {
+          }
+        case .presenceChanged(let presenceChange):
+          print("The channel \(presenceChange.channel) has an updated occupancy of \(presenceChange.occupancy)")
+          for action in presenceChange.actions {
+            switch action {
+            case let .join(uuids):
+              print("The following list of occupants joined at \(presenceChange.timetoken): \(uuids)")
+            case let .leave(uuids):
+              print("The following list of occupants left at \(presenceChange.timetoken): \(uuids)")
+            case let .timeout(uuids):
+              print("The following list of occupants timed-out at \(presenceChange.timetoken): \(uuids)")
+            case let .stateChange(uuid, state):
+              print("\(uuid) changed their presence state to \(state) at \(presenceChange.timetoken)")
+            }
+          }
+        case .uuidMetadataSet(let uuidMetadataChange):
+          print("Changes were made to \(uuidMetadataChange.metadataId) at \(uuidMetadataChange.updated)")
+          print("To apply the change, fetch a matching object and call uuidMetadataChange.apply(to: otherUUIDMetadata)")
+        case .uuidMetadataRemoved(let metadataId):
+          print("Metadata for the uuid \(metadataId) has been removed")
+        case .channelMetadataSet(let channelMetadata):
+          print("Changes were made to \(channelMetadata.metadataId) at \(channelMetadata.updated)")
+          print("To apply the change, fetch a matching object and call channelMetadata.apply(to: otherUUIDMetadata)")
+        case .channelMetadataRemoved(let metadataId):
+          print("Metadata for the channel \(metadataId) has been removed")
+        case .membershipMetadataSet(let membership):
+          print("A membership was set between \(membership.uuidMetadataId) and \(membership.channelMetadataId)")
+        case .membershipMetadataRemoved(let membership):
+          print("A membership was removed between \(membership.uuidMetadataId) and \(membership.channelMetadataId)")
+        case .messageActionAdded(let messageAction):
+          print("The \(messageAction.channel) channel received a message at \(messageAction.messageTimetoken)")
+          print("This action was created at \(messageAction.actionTimetoken)")
+          print("This action has a type of \(messageAction.actionType) and has a value of \(messageAction.actionValue)")
+        case .messageActionRemoved(let messageAction):
+          print("The \(messageAction.channel) channel received a message at \(messageAction.messageTimetoken)")
+          print("A message action with the timetoken of \(messageAction.actionTimetoken) has been removed")
+        case .subscribeError(let error):
+          print("The following error was generated during subscription \(error.localizedDescription)")
+          print("If a `disconnectedUnexpectedly` also occurred then subscription has stopped, and needs to be restarted")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 121 characters](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=5691345)